index.js

```diff
@@ -244,9 +294,9 @@ function getGermanCars(/* code here */) {
  *   return num * 2
  * }
 */
-const sum = null; // code here!
-const addFive = null; // code here!
-const argTimesTwo = null; // code here!
+const sum =(a,b)=>{return a+b;}; // code here!
+const addFive = (num)=>{return num+5;}; // code here!
+const argTimesTwo = (num)=>{return num*2;}; // code here!
```

Comment:
For one line arrow functions of this sort, you can actually ignore the return statement and the curly braces. 
`const sum = (a,b) => a + b`